content/intro-to-reading-data.md

```diff
@@ -17,21 +17,28 @@
 
 - **Accounts** are like the files in Solana’s network ledger. All state data is stored in an account. Accounts can be used for many things, but for now we’ll focus on the aspect of accounts which store SOL.
 - **SOL** is the name of Solana’s native token.
-- **Lamports** are fractional SOL and are named after [Leslie Lamport](https://en.wikipedia.org/wiki/Leslie_Lamport).
+- **Lamports** are fractional SOL and are named after [Leslie Lamport](https://en.wikipedia.org/wiki/Leslie_Lamport). 1 Lamport = 0.000000001 SOL.
 - **Public keys**, often referred to as addresses, point to accounts on the Solana network. While you must have a specific secret key to perform certain functions within accounts, anyone can read account data with a public key.
 - **JSON RPC API**: all interactions with the Solana network happens through the [JSON RPC API](https://docs.solana.com/developing/clients/jsonrpc-api). This is effectively an HTTP POST with a JSON body that represents the method you want to call.
-- **@solana/web3.js** is an abstraction on top of the JSON RPC API. It can be installed with `npm` and allows you to call Solana methods as JavaScript functions. For example, you can use it to query the SOL balance of any account:
+- **@solana/web3.js** is an abstraction on top of the JSON RPC API. It can be installed with `npm` or `yarn` and allows you to call Solana JSON RPC API as JavaScript functions. For example, you can use it to query the SOL balance of any account:
 
     ```tsx
+    import {
+      clusterApiUrl,
+      Connection,
+      PublicKey,
+      LAMPORTS_PER_SOL,
+    } from "@solana/web3.js";
+
     async function getBalanceUsingWeb3(address: PublicKey): Promise<number> {
-        const connection = new Connection(clusterApiUrl('devnet'));
+        const connection = new Connection(clusterApiUrl('devnet'), 'confirmed');
         return connection.getBalance(address);
     }
 
-    const publicKey = new PublicKey('7C4jsPZpht42Tw6MjXWF56Q5RQUocjBBmciEjDa8HRtp')
+    const publicKey = new PublicKey('7C4jsPZpht42Tw6MjXWF56Q5RQUocjBBmciEjDa8HRtp');
     getBalanceUsingWeb3(publicKey).then(balance => {
-        console.log(balance)
-    })
+        console.log(`${balance/LAMPORTS_PER_SOL} SOL`)
+    });
```

Comment:
Not your fault (this is old code), but the formatting is odd here - can you check they're using 2-spaces per CONTRIBUTING? I normally just cut and paste into a text file, save and let prettier do it's work, then paste back.